crl/idp/idp_test.go

```diff
@@ -0,0 +1,40 @@
+package idp
+
+import (
+	"encoding/hex"
+	"testing"
+
+	"github.com/letsencrypt/boulder/test"
+)
+
+func TestMakeUserCertsExt(t *testing.T) {
+	t.Parallel()
+	dehex := func(s string) []byte { r, _ := hex.DecodeString(s); return r }
```

Comment:
I know this is just copy-paste from `issuance/crl_test.go` but we should avoid throwing away errors.

```suggestion
)

func dehex(s string) []byte {
	r, err := hex.DecodeString(s)
	if err != nil {
		panic(fmt.Sprintf("failed to decode hex string: %v", err))
	}
	return r
}

func TestMakeUserCertsExt(t *testing.T) {
	t.Parallel()
```

Reply:
We're only calling dehex on static inputs, not on any dynamic values, so we know ahead of time that it's not going to return an error here.